src/locales/jp.json

```diff
@@ -0,0 +1,172 @@
+            "hide": "ヤックヤックを隠す",
+            "hide_others": "他を隠す",
+            "show": "すべてを表示",
+            "inspector": "インスペクターを開く",
+            "logout": "ロッグアウト",
+            "quit": "終了"
+        },
+        "edit": {
+            "title": "編集",
+            "undo": “元に戻す”,
+            "redo": "やり直し",
+            "paste": "ペースト",
+            "cut": "カット",
+            "copy": "コピー",
+            "copy_image_link": “イメージのリンクをコピー",
+            "copy_link": "リンクをコピー",
+            "copy_image": "イメージをコピー",
+            "select_all": "すべてを選択",
+            "language": "言語 (Language)"
+        },
+        "view": {
+            "title": "表示",
+            "emoji": "絵文字に変換",
+            "hide_dock": "ドックのアイコンを隠す",
+            "fullscreen": “全画面表示”,
+            "tray": {
+                "show_tray": "アイコンを表示",
+                "show_window": “ウインドウを表示",
+                "toggle_minimize": "アイコンに隠す",
+                "start_minimize": "起動時アイコンに隠す",
+                "close": "アイコンに閉じる"
+            },
+            "escape": {
+                "title": "escキーの行動",
+                "hide": "ウインドウを隠す",
+                "clear": "入力を消す",
+                "default": "アイコンを表示しない時の標準"
+            },
+            "conversation": {
+                "title": “チャットリスト",
+                "timestamp": "チャット日時を表示",
+                "last": "最後のチャットを表示",
+                "previous": "前のチャット",
+                "next": "次のチャット",
+                "select": "チャットを選択",
+                "thumbnails": {
+                    "show": "サムネイルを表示",
+                    "only": "サムネイルのみを表示",
+                    "animated": "動画サムネイルを表示"
+                }
+            },
+            "notification": {
+                "title": "通知",
+                "show": "通知を表示",
+                "toggle": "通知　入／切",
+                "message": "通知にメッセージを表示",
+                "username": "通知にユーザー名を表示",
+                "mute": "通知音を消音",
+                "avatar": "通知にアバター",
+                "icon": "通知にヤックヤックのアイコンを表示",
+                "custom_sound": "通知の際ヤックヤックの音を使用"
+            },
+            "color_scheme": {
+                "title": "カラー設定",
+                "default": "標準",
+                "blue": "青",
+                "dark": "暗い",
+                "material": "生地"
+            },
+            "font": {
+                "title": "フォント",
+                "extra_small": "極小",
+                "small": "小",
+                "medium": "中",
+                "large": "大",
+                "extra_large": "極大"
+            },
+            "zoom": {
+                "in": "ズームイン",
+                "out": "ズームアウト",
+                "reset": "標準"
+            }
+        },
+        "help": {
+            "title": "ヘルプ",
+            "about": {
+                "title": "ヤックヤックについて",
+                "newer": "新しいバージョンが利用可能です(%s から a %s)。",
+                "license": "ライセンス",
+                "authors": "主な作者",
+                "contributors": "貢献者"
```

Comment:
There's a string missing:

"startup": "Open on startup"

See line 163 of https://github.com/yakyak/yakyak/blob/a16f24c09fffa4f8d6596f4258b47d4d1c382f92/src/locales/en.json.template#L163